tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -74,7 +74,8 @@ def __init__(self, config, session, logger):
         self.config = config
         self.logger = logger
         self.session = session
-        self.aws_ses = session.client('ses', region_name=config.get('ses_region'))
+        if not self.config.get('queue_url').startswith('projects'):
```

Comment:
reuse the is_gcp_cloud function you defined in cli

Reply:
Good catch. Fixed in the next commit.